test/persist/test_failure_indices.sh

```diff
@@ -0,0 +1,124 @@
+#! /bin/bash
+
+export PATH="$GOBIN:$PATH"
+export TMHOME=$HOME/.tendermint_persist
+
+rm -rf "$TMHOME"
+tendermint init
+
+# use a unix socket so we can remove it
+RPC_ADDR="$(pwd)/rpc.sock"
+
+TM_CMD="tendermint node --log_level=debug --rpc.laddr=unix://$RPC_ADDR" # &> tendermint_${name}.log"
+DUMMY_CMD="abci-cli kvstore --persist $TMHOME/kvstore" # &> kvstore_${name}.log"
+
+
+function start_procs(){
+    name=$1
+    indexToFail=$2
+    echo "Starting persistent kvstore and tendermint"
+    if [[ "$CIRCLECI" == true ]]; then
+        $DUMMY_CMD &
+    else
+        $DUMMY_CMD &> "kvstore_${name}.log" &
+    fi
```

Comment:
I believe Circle CI is no more, we could simplify this:
```suggestion
    abci_cmd "kvstore_${name}.log" &
```

We might also want to be more specific about where we stash these files, `$PWD` is ambient in CI.